src/nomadic/start/commands.py

```diff
@@ -0,0 +1,108 @@
+import enum
+import os
+from contextlib import chdir
+from importlib.resources import files
+
+import click
+
+from nomadic.download.main import main as download_reference
+from nomadic.download.references import REFERENCE_COLLECTION
+from nomadic.util.config import default_config_path, write_config
+from nomadic.util.workspace import Workspace, init_workspace
+
+
+class Organism(enum.Enum):
+    pfalciparum = enum.auto()
+
+
+@click.argument(
+    "organism",
+    type=click.Choice(Organism),
+    required=True,
+)
+@click.option(
+    "-w",
+    "--workspace",
+    "workspace_path",
+    default="nomadic",
+    type=click.Path(exists=False),
+    help="Path of the the workspace where all the files will be stored",
+)
+@click.option(
```

Comment:
Is this still necessary if we have organism?

Reply:
I saw that we have also Dd2, so I though there might be the need to override the reference for a given organism? 
Maybe not needed.

Reply:
Let's drop for now, because I think for any given organism there is only ONE reference genome which you should map to.

Reply:
okay